README.md

```diff
@@ -56,9 +56,9 @@ export default reducer;
 
 # Documentation
 
-* [Introduction](/docs/introduction)
-* [API](/docs/api)
-* [Middleware](/docs/middleware)
-* [External Resources](/docs/ExternalResources.md)
-* [Change Log](/docs/ChangeLog.md)
-* [Contributors](/docs/Contributors.md)
```

Comment:
I guess this pointed back to the repo; now this will just point to the documentation site. Fine by me.